datasets/crows_pairs/README.md

```diff
@@ -0,0 +1,140 @@
+---
+annotations_creators:
+- crowdsourced
+language_creators:
+- crowdsourced
+languages:
+- en
+licenses:
+- cc-by-sa-4.0
+multilinguality:
+- monolingual
+size_categories:
+- 1K<n<10K
+source_datasets:
+- original
+task_categories:
+- sequence-modeling
+task_ids:
+- language-modeling
```

Comment:
Bit unsure about the task tag. The dataset is created to measure the degree to which U.S. stereotypical biases present in the MLMs, so using `language-modeling` tag here. Wdyt @lhoestq @yjernite ?

Reply:
maybe add another task ?
```
task_categories:
- sequence-modeling
task_ids:
- language-modeling
- sequence-modeling-other-social-biais-detection
```

Reply:
@patil-suraj there's also a ``text-scoring` task category which would be useful there, you can add `other-bias-evaluation` as a specific task